.prettierignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
we dont use prettier? should be removed?!

Reply:
We [have it in the main repo](https://github.com/fastify/fastify/blob/345c85ed2a7d9a1034d2c19a1d4b8cbcf21af1e4/.prettierignore), so I guess that's where this has come from.

Reply:
ok, it is also in the skeleton ;)

Reply:
Actually it's there because I wanted to quickly disable prettier which I have enabled as a plugin in vscode, and I shouldn't have committed it. I'll remove it from the PR.